config/config.js

```diff
@@ -44,7 +45,18 @@ module.exports = {
       name: 'Only Show Magic Results',
       description:
         'When checked, strings searched that do not immediately have a Magic suggestion will not be displayed in the overlay.',
-      default: false,
+      default: true,
+      type: 'boolean',
+      userCanEdit: true,
+      adminOnly: false
+    },
+    {
+      key: 'ignoreEntityTypes',
+      name: 'Ignore Entity Types',

```

Comment:
@penwoodjon Not sure how tough this would be to do but would be nice for this to be a multi-select where selected types are ignored (rather than being an all or nothing boolean).

I know we're on tight deadline for this though so happy to have this as is and make that a future feature if you think it'll add too much complexity.